prov/verbs/src/verbs_init.c

```diff
@@ -493,6 +493,18 @@ int vrb_find_max_inline(struct ibv_pd *pd, struct ibv_context *context,
 	}
 	qp_attr.sq_sig_all = 1;
 
+	if (vrb_gl_data.def_inline_size >= max_inline) {
+		qp_attr.cap.max_inline_data = vrb_gl_data.def_inline_size;
+		qp = ibv_create_qp(pd, &qp_attr);
+		if (qp) {
+			rst = vrb_gl_data.def_inline_size;
+			goto out;
+		}
+
+		rst = max_inline = vrb_gl_data.def_inline_size;
```

Comment:
Need to start over from the next do loop instead. Use `vrb_gl_data.def_inline_size` in the condition of the matching `while`.